### ISSUE TITLE
READ FIRST: NOTICE ABOUT ISSUES

### ISSUE THREAD

**Issue by joshuag1000** [flathub/cc.arduino.IDE2#35]
The Flathub release is an 3rd party wrapper maintained by the community and is not an officially supported method of installing the IDE.
This does **not** mean that I (the current maintainer) and the Arduino Dev's are not interested in your issues.

If you experience **any** issues with the IDE please report them here first so that I can determine if this is a flatpak issue or an IDE issue and pass it onto the Arduino Devs.

Alternatively you can download an officially supported version of the IDE at https://www.arduino.cc/en/software and test to see if the issue is a flatpak issue yourself or if it truly is an IDE issue.
If your issue is a genuine IDE issue then please report it at: https://github.com/arduino/arduino-ide/issues/
If your issue is a flatpak specific issue please report it at: https://github.com/flathub/cc.arduino.IDE2/issues/

Why is it like this?

Well to keep it simple for everyone. 
With the flathub release being an unofficial wrapper there is no official support. To ensure that the best support and experience is given having all IDE issues in regards to the flathub release dealt with here first means that the dev teams don't have to deal with issues that were caused by this release. 
Furthermore, by having all issues (related to the flathub release) here I don't have to search through the IDE's issues to find misplaced issues meaning that i can respond to your issues much quicker.

Any questions regarding this should be posted here. Any issues should be created here: https://github.com/flathub/cc.arduino.IDE2/issues/

TLDR: 
- Check to ensure your issue hasn't been reported at: https://github.com/flathub/cc.arduino.IDE2/issues/
- Report the issue at: https://github.com/flathub/cc.arduino.IDE2/issues/new
